projectq/ops/_gates.py

```diff
@@ -117,6 +117,21 @@ def __str__(self):
 Tdag = Tdagger = get_inverse(T)
 
 
+class SqrtXGate(BasicGate):
+    """ Square-root X gate class """
+    @property
+    def matrix(self):
+        return 0.5 * np.matrix([[1+1j, 1-1j], [1-1j, 1+1j]])
+
+    def tex_str(self):
+        return r'$\sqrt{\neg}$'
```

Comment:
In my opinion, instead of \neg (¬), it looks more natural to have X,
i.e. $\sqrt{X}
What do you or @thomashaener think?

Reply:
:+1: I think \sqrt{X} is proper, \sqrt{\neg} is copied from wiki, which is less intuitive. In literature, they are used in equal frequency. We can insist on \sqrt{X}.